src/eventshub/tailwind.config.ts

```diff
@@ -1,13 +1,10 @@
 /** @type {import('tailwindcss').Config} */
 export default {
   presets: [require('decanter')],
+  prefix: 'er-',
```

Comment:
@rebeccahongsf bad news, I opted to prefix our output because of a clash in classnames with the base theme. eg: centered-container. I looked into removing just the ones that hit hard but you cannot remove plugins from presets so I was unable to go through route. This seemed like the path of least resistance.